src/traits/mod.rs

```diff
@@ -80,7 +80,17 @@ pub trait Group:
   fn preprocessed(&self) -> Self::PreprocessedGroupElement;
 
   /// Produce a vector of group elements using a static label
-  fn from_label(label: &'static [u8], n: usize) -> Vec<Self::PreprocessedGroupElement>;
+  fn from_label(label: &'static [u8], n: usize) -> Vec<Self::PreprocessedGroupElement> {
+    Self::from_label_with_offset(label, 0, n)
+  }
+
+  /// Produce a vector of `n - offset` group elements using a static label,
+  /// by first generating `n` elements and then discarding the first `offset` of them
```

Comment:
This feels like it makes an assumption those elements are *deterministically* generated from the label, and this should be documented very explicitly for the offset to make sense.